### ISSUE TITLE
Mouse over text on the hand signals

### ISSUE THREAD

**Issue by pauldougan** [LeePorte/hand-signals#1]
![image](https://user-images.githubusercontent.com/34273121/80597628-007d0000-8a20-11ea-9185-6db642da0a6e.png)

Can we have some explanatory mouse over text to explain the hand signals please!

**Comment by LeePorte** [comment]
I'll certainly have an investigate, shouldn't be too hard to sort

**Comment by LeePorte** [comment]
Hi @pauldougan I have added some tool tips and alt text to the images. this has been built and pushed to the chrome web store. it should be visible as version 0.0.3 shortly.
